integrations/http/server.go

```diff
@@ -12,28 +12,44 @@ import (
 	"go.autokitteh.dev/autokitteh/internal/kittehs"
 	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+	"go.autokitteh.dev/autokitteh/web/static"
 )
 
+const (
+	// Save new autokitteh connections with user-submitted secrets.
+	uiPath = "/httprest/connect/"
```

Comment:
a though, let me know what do you think,
maybe if this is a rest supporting integration
it should be just /http
get will get the ui
post will save a new connection ?

Reply:
See my comment above about why I wanted a different path from `http`.

I don't think your idea will work because I also use POST - when the user loads the web page in `/httprest/connect`, it contains a web form, which the user submits as a POST to `/httprest/save` (which points to my webhook in `save.go`).

Reply:
maybe all integrations should start with /integrations/<integration-name>/ ? 
and all web hooks should be received at /webhooks ?

Reply:
we might opt to give a root mux to all integrations where they will build on top of it. we should give them an `/integrations/{integration_name}/` root mux imo.

Reply:
done - see https://github.com/autokitteh/autokitteh/pull/160#discussion_r1564591312